pinot-core/src/main/java/org/apache/pinot/core/data/manager/realtime/LLRealtimeSegmentDataManager.java

```diff
@@ -1465,9 +1465,10 @@ private void makeStreamConsumer(String reason) {
     if (_partitionGroupConsumer != null) {
       closePartitionGroupConsumer();
     }
-    _segmentLogger.info("Creating new stream consumer, reason: {}", reason);
+    _segmentLogger.info("Creating new stream consumer for topic partition {} , reason: {}", _clientId, reason);
```

Comment:
`_segmentLogger` already contains segment name (which includes partitionGroupId). The segment name also includes the table name (which implies the topic name). So, do you really need to add the clientId here?

Reply:
I think it is useful to keep the log messages as clear as possible, without assumptions about the reader knowing the format of segment name or the fact that the table name and topic name are the same.  I would leave the clientId here, although I would really like to explicitly call out the topic and partition :)